Cargo.lock

```diff
@@ -5899,13 +5899,24 @@ dependencies = [
 
 [[package]]
 name = "hostname"
-version = "0.4.1"
+version = "0.3.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3c731c3e10504cc8ed35cfe2f1db4c9274c3d35fa486e3b31df46f068ef3e867"
+dependencies = [
+ "libc",
+ "match_cfg",
+ "winapi",
+]
+
+[[package]]
+name = "hostname"
+version = "0.4.0"
```

Comment:
We used `0.4.1` prior to this PR but use `0.4.0` + `0.3.1` now. Is it expecged?

Reply:
The PR doesn't sync up with the main branch yet. Let me update it.